spec/presenters/sufia/collection_presenter_spec.rb

```diff
@@ -8,14 +8,19 @@
                             :language, :identifier, :based_near, :related_url] }
   end
 
-  let(:collection) { Collection.new(id: '111', description: 'a nice collection', title: 'A clever title') }
-  let(:solr_doc) { SolrDocument.new(collection.to_solr) }
+  let(:collection) { Collection.new(id: '111', description: 'a nice collection', title: 'A clever title', resource_type: ['Collection']) }
   let(:ability) { double }
   let(:presenter) { described_class.new(solr_doc, ability) }
+  let(:solr_doc) { SolrDocument.new(collection.to_solr, resource_type: 'Collection') }
```

Comment:
is it necessary to provide `resource_type: 'Collection'` here?  Shouldn't `collection.to_solr` take care of that?